services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/DigitalGreenCertificateStructureProvider.java

```diff
@@ -63,20 +63,14 @@ public DigitalGreenCertificateStructureProvider(DistributionServiceConfig distri
    * Business rules Cbor encoded structures.
    */
   public DirectoryOnDisk getDigitalGreenCertificates() {
-    try {
-      return constructArchiveToPublish(distributionServiceConfig.getDigitalGreenCertificate());
-    } catch (UnableToLoadFileException e) {
-      logger.error("We don't generate a value-sets file and this shouldn't override existing ones.", e);
-      return new DirectoryOnDisk(EMPTY_STRING);
-    }
+    return constructArchiveToPublish(distributionServiceConfig.getDigitalGreenCertificate());
   }
 
-  private DirectoryOnDisk constructArchiveToPublish(DigitalGreenCertificate dgcConfig)
-      throws UnableToLoadFileException {
+  private DirectoryOnDisk constructArchiveToPublish(DigitalGreenCertificate dgcConfig) {
```

Comment:
Maybe we can find a better naming for the methodname and the parameter. Or is DigitalGreenCertificate a config file?

Reply:
DigitalGreenCertificate is a config. For me the method name seems okay. Do you have any suggestion ?

Reply:
no it is alright for me :)